wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/rapidreactcommandbot/subsystems/Drive.java

```diff
@@ -60,6 +83,13 @@ public Drive() {
     // Sets the distance per pulse for the encoders
     m_leftEncoder.setDistancePerPulse(DriveConstants.kEncoderDistancePerPulse);
     m_rightEncoder.setDistancePerPulse(DriveConstants.kEncoderDistancePerPulse);
+
+    // Set the controller to be continuous (because it is an angle controller)
+    m_controller.enableContinuousInput(-180, 180);
```

Comment:
Add a note that not all gyros return (-180,180) and that this may need to be adjusted accordingly

Reply:
Does that matter? A gyro that returns (0, 360) will have the value be converted to be within (-180, 180). And I'm pretty sure there's actually no difference between calling `enableContinuousInput` with either (-180, 180) or (0, 360).

Reply:
Where would the [0, 360] be converted to [-180, 180]? What about gyros that don't wrap and keep going past 360?

Reply:
Both ProfiledPIDController and PIDController will take the min and max terms and convert them to an `errorBound` using this line:
```java
double errorBound = (m_maximumInput - m_minimumInput) / 2.0;
```
The remaining calculations are based on this `errorBound`. No matter if you plug in (0, 360) or (-180, 180), you will always end up with `errorBound = 180`.

As for gyros that don't wrap, that's handled by the call to MathUtil.inputModulus, which will place the angle within the specified bounds.